src/FhirLoader.Common/BundleClient.cs

```diff
@@ -87,7 +89,7 @@ public async Task Send(ProcessedBundle bundle, Action<int, long>? metricsCallbac
                 catch (JsonReaderException)
                 {
                     _logger.LogError(responseString);
-                }   
+                }
```

Comment:
Can we rename this class `FhirResourceClient` instead of `BundleClient`?

Reply:
Renamed as FhirResourceClient.